tests/phpunit/TestFramework/PhpUnit/CommandLine/ArgumentsAndOptionsBuilderTest.php

```diff
@@ -40,33 +40,41 @@
 
 final class ArgumentsAndOptionsBuilderTest extends TestCase
 {
-    public function test_it_builds_correct_command(): void
+    /**
+     * @var ArgumentsAndOptionsBuilder
+     */
+    private $builder;
+
+    protected function setUp(): void
+    {
+        $this->builder = new ArgumentsAndOptionsBuilder();
+    }
+
+    public function test_it_can_build_the_command_without_extra_options(): void
     {
         $configPath = '/config/path';
-        $builder = new ArgumentsAndOptionsBuilder();
 
         $this->assertSame(
             [
                 '--configuration',
                 $configPath,
-                '--verbose',
-                '--debug',
             ],
-            $builder->build($configPath, '--verbose --debug')
```

Comment:
Why the second parameter is emptied?

Reply:
It's a swap around: I moved the "without extra options" as the first one